src/main/navigation/navigation.c

```diff
@@ -3226,7 +3226,7 @@ navArmingBlocker_e navigationIsBlockingArming(bool *usedBypass)
     }
 
     // Don't allow arming if any of NAV modes is active
-    if (!ARMING_FLAG(ARMED) && navBoxModesEnabled && !navLaunchComboModesEnabled) {
+    if (!ARMING_FLAG(ARMED) && !FLIGHT_MODE(FAILSAFE_MODE) && navBoxModesEnabled && !navLaunchComboModesEnabled) {
```

Comment:
I would rather tweak the UI, not the logic, WDYT?

Reply:
Or we might just refactor the RX input not to present 1500 when in FS. Toughts?

Reply:
I would simply change the priorities of OSD messages. Internally we have both `NAV_IS_ALREADY_ACTIVE` and `NO_RC_LINK` triggered, the issue is only which one we want to show to the user first

Reply:
I guess NO_RC_LINK should have the priority

Reply:
I tried to swap around messages in osdArmingDisabledReasonMessage (osd.c) giving priority to 
RC LINK
ACC/MAG not calibrated
but still it displays navigation unusafe. 
Unsure, yet, what it is going on there

Reply:
`osdArmingDisabledReasonMessage` merely displays the message. The actual reason (and only one reason) is returned by `isArmingDisabledReason` in runtime_config.c and that returns the first arming blocking bit set in `armingFlag_e`. 

You should probably swap bits for `ARMING_DISABLED_RC_LINK` (bit 18) and `ARMING_DISABLED_NAVIGATION_UNSAFE` (bit 11).